python/cudf/groupby/legacy_groupby.py

```diff
@@ -454,16 +454,14 @@ def _get_function(x):
     _auto_generate_grouper_agg(locals())
 
     def apply(self, function):
-        """Apply a transformation function over the grouped chunk.
+        """Apply a python transformation function over the grouped chunk.
 
-        This uses numba's CUDA JIT compiler to convert the Python
```

Comment:
Why are these lines being deleted? AFAIK Numba is still used for UDFs.

Reply:
I guess I should read the description before reviewing the code changes. :) I'm not the right reviewer for python docs, by a long stretch.